kernel/mlx_ethernet/src/command_queue.rs

```diff
@@ -398,8 +398,8 @@ pub enum QueryHcaCapMaxOpMod {
 /// Possible values of the opcode modifer when the opcode is [`CommandOpcode::QueryHcaCap`] and we want to retrieve current values of capabilities.
 #[derive(Copy, Clone)]
 pub enum QueryHcaCapCurrentOpMod {
-    GeneralDeviceCapabilities       = (0x0 << 1) | 0x1,
-    EthernetOffloadCapabilities     = (0x1 << 1) | 0x1
+    GeneralDeviceCapabilities       =              0x1,
```

Comment:
i think this is important to leave here in that it's reflecting the logic of how the const values of these hardware flags are calculated. So let's leave this one as is, and allow the `identity_op` just for this.